tools/cldr-code/src/main/java/org/unicode/cldr/test/FlexibleDateFromCLDR.java

```diff
@@ -30,7 +30,7 @@
  *
  * @author markdavis
  */
-class FlexibleDateFromCLDR {
+public class FlexibleDateFromCLDR {
```

Comment:
There's an old (?) comment above, "Temporary class while refactoring", which casts suspicion on making this public